apiserver/facade/facadetest/modelcontext.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package facadetest
+
+import (
+	"context"
+
+	"github.com/juju/juju/core/objectstore"
+	"github.com/juju/juju/internal/servicefactory"
+)
+
+// ModelContext implements facade.ModelContext in the simplest possible way.
+type ModelContext struct {
+	Context
+
+	ServiceFactoryForModel_ servicefactory.ServiceFactory
+	ObjectStoreForModel_    objectstore.ObjectStore
+}
+
+// ServiceFactoryForModel returns the services factory for a given
+// model uuid.
+func (c ModelContext) ServiceFactoryForModel(modelUUID string) servicefactory.ServiceFactory {
```

Comment:
Should something be done with this argument, or should it at least be validated? Right now it is just ignored.

Reply:
It's a test stub, the best thing would to just remove the name and use underscore